Adding 2 test files

hanyelgaml-roche opened the pull request.

Commit:
Adding 2 test files

Commit:
Fixing the following Problem:
when calling `bins_rows`, reformat specified columns to floats otherwise computing mean fails

jkanche commented on src/efs_parser/BaseFile.py:
```diff
@@ -219,12 +219,19 @@ def bin_rows(self, data, chr, start, end, columns=None, metadata=None, bins = 40
             if len(temps) > 0:
                 for col in columns:
-                    row[col] = float(np.mean(temps[col].values))
+                    #row[col] = float(np.mean(temps[col].values))
+                    row[col] = np.mean(self.change_to_float(temps[col].values))
```
@hanyelgaml-roche 

use the pandas astype method instead - https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.astype.html#pandas-dataframe-astype

Commit:
removing the function
def change_to_float(self,dummy):
and replacing the functionality with a single line:
values_to_fix = [float(i) for i in temps[col].values]
pandas astype did not work because temps is an array and not a DataFrame

jkanche commented on src/efs_parser/BaseFile.py:
```diff
@@ -220,18 +220,14 @@ def bin_rows(self, data, chr, start, end, columns=None, metadata=None, bins = 40
                     #row[col] = float(np.mean(temps[col].values))
-                    row[col] = np.mean(self.change_to_float(temps[col].values))
+                    #row[col] = np.mean(self.change_to_float(temps[col].values))
+                    values_to_fix = [float(i) for i in temps[col].values]
```
this doesn't work ? 

```
temps[col].apply(float)
```

or this ?

```
temps[col].astype(float64)
```

hanyelgaml-roche replied:
These 2 work:
temps[col].apply(float)
temps[col].astype(float)
However, this does work:
temps[col].astype(float64)

Thank you for the hint. I will update the code accordingly

jkanche replied:
ok, Iets use the pandas inbuilt method `astype`

```temps[col].astype(float)```

hanyelgaml-roche replied:
I have committed the change

Commit:
row[col] = np.mean(temps[col].astype(float))
